### PR TITLE
Commercial containers

### PR DESCRIPTION
**Pull request by piuccio**
Implement two new commercial containers for single and multi campaigns.

For the moment it simply renders as a fixed container, small slow 4.

Someone else should pick this up and actually implement the rendering.

@regiskuckaertz @janua

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
:metal:

File: common/app/slices/CommercialContainer.scala [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package slices
+
+object CommercialContainer {
+  val all: Map[String, Container] = Map(
+    ("commercial/single-campaign", Commercial(SingleCampaign(FixedContainers.fixedSmallSlowIV))),
+    ("commercial/multi-campaign", Commercial(MultiCampaign(FixedContainers.fixedSmallSlowIV)))
+  )
+}
+
+sealed trait CommercialContainer
```
Comment (janua) [inline common/app/slices/CommercialContainer.scala]: Not so sure about this name... Maybe `CommercialCampaign`?
The thing on the outside of this is the `Container`, while the thing inside is the `ContainerDefinition`. This middle part has never exited before but I don't think it's strictly containter!

**Comment by janua** [comment]
Great! :+1:

File: common/app/slices/CommercialContainer.scala (hunk 1 above)
Comment (piuccio) [inline common/app/slices/CommercialContainer.scala]: The thing about `campaign` is that who knows what types of commercial containers we'll have to create in the future. `campaign` is what these two containers do right now, it does not describe what they are

Comment (janua) [inline common/app/slices/CommercialContainer.scala]: Time to get out the dictionary :books:

**Pull request merged**